### DISCUSSION TITLE
switch airgap to internet repo

### DISCUSSION THREAD

**Discussion by ernstdieter** [Security-Onion-Solutions/securityonion#2239] [Unsupported Versions] [answered]
Hi guys,
has anyone an idea how I can switch the default yum repo of airgap to the default internet repo of Security Onion 2?
I installed the iso with version 2.3.2 and now I want to run updates via internet to version 2.3.10. It seems the folder /etc/yum.repo.d will be overwritten after restart. What are the steps to change it?

Thanks in advance,
Ernst

**Comment by TOoSmOotH** [comment, marked as answer]
If you download the ISO you can use that to do the update. You don't need to burn it to a CD or USB, you can just copy it over. If not, you will need to set airgap to false in the global.sls and then manually add the repos back to all nodes in the grid. It will be much easier to just use the ISO. You would need to add the default repos back as well as epel, saltstack, and docker. You can pull the docker, epel and saltstack from the setup script.

**Reply by ernstdieter** [reply]
The ISO contains systems updates as well?

BR

**Reply by TOoSmOotH** [reply]
It does.
